pkg/apis/serving/v1beta1/podspec.go

```diff
@@ -0,0 +1,226 @@
+package v1beta1
+
+import (
+	v1 "k8s.io/api/core/v1"
+)
+
+// PodSpec is a description of a pod.
```

Comment:
Can you write up why we cant use it directly?

Reply:
And what changes we made.

Reply:
updated with comments